src/cartPages/cartpages.js

```diff
@@ -1,3 +1,86 @@
+import { getRestaurants } from "../restaurants/restaurants";
+
 export function getCartTemplate() {
   return ` <h1>Cart</h1>`;
 }
+
+export const displayItemsFromLocalStorage = () => {
+  const cart = JSON.parse(localStorage.getItem("cart")) || [];
+
+  const cartItemsHtml = cart
+    .map((cartItem) => {
+      const item = cartItem.item;
+      return `
+      <tr>
+        <td>${item.name} &nbsp;</td>
+        <td>${item.topping} &nbsp;</td>
+        <td>$${item.price.toFixed(2)}</td>
+        <td><button data-id="${
+          cartItem.id
+        }" class="remove-item-btn">Remove</button></td>
+      </tr>
+    `;
+    })
+    .join("");
+
+  const totalPrice = cart.reduce(
+    (acc, cartItem) => acc + cartItem.item.price,
+    0
+  );
+
+  const cartTemplate = `
+    <h1>Cart</h1>
+    <table>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Toppings</th>
+          <th>Price</th>
+          <th></th>
+        </tr>
+      </thead>
+      <tbody>
+        ${cartItemsHtml}
+      </tbody>
+      <tfoot>
+        <tr>
+          <td colspan="3">Total</td>
+          <td>$${totalPrice.toFixed(2)}</td>
+        </tr>
+      </tfoot>
+    </table>
+    <br>
+    <div style="text-align: center;">
+      <button id="checkout-btn">Checkout</button>
+    </div>
+  `;
+  document.getElementById("main").innerHTML = cartTemplate;
+
+  const removeItemBtns = document.querySelectorAll(".remove-item-btn");
```

Comment:
This variable doesn't seem to be needed.
This section can be a separate function and you can send parameters if you need something from this function.
Try to make it a pure function.